RELEASE-NOTES.md

```diff
@@ -1,5 +1,39 @@
 # Prefect Release Notes
 
+## Release 2.10.5
+
+### Deploy a Prefect flow via Github Actions
+
+With the new [Deploy a Prefect flow](https://github.com/marketplace/actions/deploy-a-prefect-flow) Github action, you can automate the build process for deployments orchestrated by Prefect Cloud. The action leverages the new [Projects](https://docs.prefect.io/latest/concepts/projects/) system. See the [action page](https://github.com/marketplace/actions/deploy-a-prefect-flow) for examples and configuration options.
```

Comment:
```suggestion
With the new [Deploy a Prefect flow](https://github.com/marketplace/actions/deploy-a-prefect-flow) Github Action, you can automate the build process for deployments orchestrated by Prefect Cloud. The action leverages the new [Projects](https://docs.prefect.io/latest/concepts/projects/) system. See the [action page](https://github.com/marketplace/actions/deploy-a-prefect-flow) for examples and configuration options.
```
Not totally sure if a singular GitHub Action should be capitalized 🤷‍♀️

Reply:
Good catch. GitHub should be camel case too.